PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebResult.kt

```diff
@@ -0,0 +1,13 @@
+package com.paypal.android.checkoutweb
+
+import android.net.Uri
+import androidx.annotation.NonNull
+import org.json.JSONObject
+
+class PayPalWebResult internal constructor(@NonNull uri: Uri, @NonNull metadata: JSONObject) {
```

Comment:
This name is a little confusing now - maybe like `PayPalDeepLinkUrlResult`? It's internal so can be renamed in the future but just for clarity now